feature_extraction.py

```diff
@@ -196,6 +204,63 @@ def save(self, filepath):
         joblib.dump(self.target_encoder, filepath)
 
 
+class TextCounter(BaseTransformer):
```

Comment:
@pknut I think this class is available in steps.preprocessing which means you can simply override what you need to override or use it as is.